src/policy/policy.cpp

```diff
@@ -183,16 +183,22 @@ bool IsStandardTx(const CTransaction& tx, const std::optional<unsigned>& max_dat
  *    as potential new upgrade hooks.
  *
  * Note that only the non-witness portion of the transaction is checked here.
+ *
+ * We also check the total number of sigops across the whole transaction.
```

Comment:
Could add: `The per-transaction sigop limit introduced in BIP54 does not cover the witness.`

Reply:
Don't you think it would be confusing to point here since this function does not concern itself with the witness? Especially as it's pointed right before.

Reply:
Well that's what confused me. My first thought was: why does this work if we're not checking the witness?

But maybe the sigops related comment needs to go above the "only the non-witness" portion.

Reply:
Sounds fine, forgot to push this?

Reply:
Changed to "non-witness" and actually pushed it this time. :)